src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -3371,6 +3375,24 @@ int Player_CanUpdateItems(Player* this) {
             (Player_ItemToItemAction(this->heldItemId) == this->heldItemAction));
 }
 
+/**
+ * Updates the Upper Body system.
+ * The Upper Body system is composed of an upper action function and
+ * a seperate skelanime that can play an animation which is different
+ * from the main skelanime.
+ *
+ * @return true if the upper body is "busy", false otherwise.
+ *
+ * The upper body being "busy" can mean a few things:
+ * - Hookshot has just connected with something that Player can fly to
```

Comment:
"fly to" sounds kinda weird. Maybe "hooked to" or "moved to"?

Reply:
original anim name, fwiw: `gPlayerAnim_link_hook_fly_start`
but yeah i would have used the word fly anyway, regardless of their term